lib/internal/Magento/Framework/Image/Adapter/Gd2.php

```diff
@@ -3,13 +3,19 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Framework\Image\Adapter;
 
 /**
  * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
  */
 class Gd2 extends \Magento\Framework\Image\Adapter\AbstractAdapter
 {
+    /**
+     * Keep the transparency for the images in the wysiwyg editors
+     */
+    const KEEP_TRANSPARENCY = true;
```

Comment:
I think we don't need constant fir true value in this class, just assign true to property when it needed

Reply:
Updated it, thanks.